### ISSUE TITLE
[WORK-AROUND]  Dir name incorrect for DKMS install script after unzipping files

### ISSUE THREAD

**Issue by ckrzen** [aircrack-ng/rtl8812au#264]
Dir name is wrong after unzipping downloaded files:

```
~/Downloads/rtl8812au-5.2.20$ sudo ./dkms-install.sh 

About to run dkms install steps...
cp: cannot stat '../rtl8812au': No such file or directory
Error! Could not find module source directory.
Directory: /usr/src/rtl8812au-5.2.20.2 does not exist.
Error! Could not find module source directory.
Directory: /usr/src/rtl8812au-5.2.20.2 does not exist.
Error! Could not find module source directory.
Directory: /usr/src/rtl8812au-5.2.20.2 does not exist.
Finished running dkms install steps.
```

WORK-AROUND:

```
~/Downloads/rtl8812au-5.2.20$ cd ..

~/Downloads$ mv rtl8812au-5.2.20 rtl8812au

~/Downloads$ cd rtl8812au/

~/Downloads/rtl8812au$ sudo ./dkms-install.sh 

About to run dkms install steps...

Creating symlink /var/lib/dkms/rtl8812au/5.2.20.2/source ->
                 /usr/src/rtl8812au-5.2.20.2

DKMS: add completed.

Kernel preparation unnecessary for this kernel.  Skipping...

Building module:
cleaning build area...
'make' -j4 KVER=4.15.0-43-generic KSRC=/lib/modules/4.15.0-43-generic/build............
cleaning build area...

DKMS: build completed.

```

**Comment by kimocoder** [comment]
Thanks. I'll add a fix shortly!

**Comment by kimocoder** [comment]
Fixed and updated in both v5.2.20 and v5.3.4 :1st_place_medal:
